3d/splitflap.scad

```diff
@@ -100,6 +100,7 @@ num_flaps = 40;
 
 flap_hole_radius = (flap_pin_width + 1) / 2;
 flap_gap = 1;
+function get_flap_gap() = flap_gap;
```

Comment:
Ah interesting; is this a workaround to expose the flap_gap variable when `use`ing splitflap.scad rather than `include`ing it? If so, mind adding a comment to that effect here for clarity?

Reply:
Yes, exactly. Resolved with b524897.